packages/tailwind/src/utils/config-handler.ts

```diff
@@ -57,15 +63,15 @@ const generateConfig = <T extends ThemeOptions>(theme: T, parsedColors: ParsedCo
 
   const colors = () =>
     ([] as StyleHandler[]).concat(
-      ...AVAILABLE_VARIANTS.map((variant) => toColorsStyles(parsedColors, variant))
+      ...AVAILABLE_VARIANTS.map((variant) => toColorsStyles(parsedColors, variant, prefixToAdd))
```

Comment:
The suggestion above is because it's a little bit hard to understand how, why and where prefix became prefixToAdd